src/WinRPM.jl

```diff
@@ -463,7 +463,13 @@ function do_install(package::Package)
         write(f, data[1])
     end
     info("Extracting: ", name)
-    cpio = splitext(path2)[1]*".cpio"
+
+    @static if VERSION < v"0.7.0-DEV"
```

Comment:
this should be specific to the merge commit that upgraded 7zip

Reply:
I agree. but my newness to Julia means I need some help here.  Is there an easy way to find that other than digging through hundreds of commits?

Reply:
there's a commit-name script in contrib of a julia checkout, run it on https://github.com/JuliaLang/julia/commit/8153470976b056c71b325f3fa1c1e9fe10aa27df